tool/tsh/tsh.go

```diff
@@ -1471,3 +1507,51 @@ func reissueWithRequests(cf *CLIConf, tc *client.TeleportClient, reqIDs ...strin
 	}
 	return nil
 }
+
+func getLatencyProfileFile(path string) (*os.File, error) {
+	var fullPath string
+	timeStamp := time.Now().Format("2006-01-02_15:04:05")
+	suffix := fmt.Sprintf("latency_profile_%s.txt", timeStamp)
+	if path != "" {
+		if _, err := os.Stat(path); err != nil {
+			if os.IsNotExist(err) {
+				// Make all if it does not exist
+				err = os.MkdirAll(path, 0700)
+				if err != nil {
+					fmt.Fprintln(os.Stderr, utils.UserMessageFromError(err))
+					return nil, err
+				}
+			} else {
+				fmt.Fprintln(os.Stderr, utils.UserMessageFromError(err))
+				return nil, err
+			}
+
+		}
+		// Remove trailing slash
+		if path[len(path)-1] == '/' {
+			path = path[0 : len(path)-1]
+		}
+		fullPath = filepath.Join(path, suffix)
+	} else {
+		dir, err := os.Getwd()
+		if err != nil {
+			return nil, err
+		}
+		fullPath = filepath.Join(dir, suffix)
+	}
+	fo, err := os.Create(fullPath)
+	if err != nil {
+		fmt.Printf("%v", err)
+		fmt.Fprintln(os.Stderr, utils.UserMessageFromError(err))
+		return nil, err
+	}
+	return fo, nil
+}
+
+func writeHistogram(w io.Writer, h *hdrhistogram.Histogram, ticks int32, scale float64) error {
```

Comment:
this func looks like a thin wrapper around `h.PercentilesPrint`, let's remove it and call `h.PercentilesPrint` directly